sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Customizations/Models/TelemetryItem.cs

```diff
@@ -74,6 +74,7 @@ internal partial class TelemetryItem
             Tags[ContextTagKeys.AiCloudRoleInstance.ToString()] = telemetryItem.Tags[ContextTagKeys.AiCloudRoleInstance.ToString()];
             Tags[ContextTagKeys.AiInternalSdkVersion.ToString()] = SdkVersionUtils.s_sdkVersion;
             InstrumentationKey = telemetryItem.InstrumentationKey;
+            SampleRate = telemetryItem.SampleRate;
```

Comment:
Do we want to call this "sample rate" or "sample ratio"?
My understanding is that "sample rate" is an integer number for frequency based sampling (e.g. an audio stream has a sample rate of 44kHz), "sample ratio" is a floating point between [0, 1] which represents the actual samples taken divided by the total number of observable samples.

Reply:
Excellent point, but I think our hands are tied on this one.

`SampleRate` comes from Breeze's schema.
https://github.com/Azure/azure-rest-api-specs/blob/e88242fc7da7860d36ab4a1700e3404a51d53eeb/specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15_Preview/swagger.json#L855-L859

Reply:
Customer will set sample ratio in the `TracerProvider`. Our backend schema accepts only `sampleRate`, we have to convert ratio to percentage and send it.

```
using var tracerProvider = Sdk.CreateTracerProviderBuilder()
                            .SetSampler(new ApplicationInsightsSampler(0.5F))
                            .AddAzureMonitorTraceExporter(o =>
                            {
                                o.ConnectionString = ConnectionString;
                            })
                            .Build();
```

Reply:
Anything we can do to make it clear? e.g. adding a comment explaining that the naming/concept is misleading, e.g. raising this to the ingestion team